en/api/events.md

```diff
@@ -215,9 +215,9 @@ Event emitted after zooming data area.
     start: number
     // percentage of zoom finish position, 0 - 100
     end: number
-    // data value of zoom start position; only exists in zoom event of triggered by toolbar

```

Comment:
I think we can't simply remove it. We may need to add some information to indicate the time when these two parameters have been supported.